prover/prover_dal/src/fri_witness_generator_dal.rs

```diff
@@ -352,23 +353,23 @@ impl FriWitnessGeneratorDal<'_, '_> {
             sqlx::query!(
                 r#"
                 INSERT INTO
-                    scheduler_witness_jobs_fri (
+                    recursion_tip_witness_jobs_fri (
                         l1_batch_number,
-                        scheduler_partial_input_blob_url,
-                        protocol_version,
                         status,
+                        number_of_final_node_jobs,
+                        protocol_version,
                         created_at,
                         updated_at
                     )
                 VALUES
-                    ($1, $2, $3, 'waiting_for_proofs', NOW(), NOW())
+                    ($1, 'waiting_for_proofs', $2, $3, NOW(), NOW())
                 ON CONFLICT (l1_batch_number) DO
                 UPDATE
                 SET
                     updated_at = NOW()
```

Comment:
I actually wonder if we shouldn't update the closed_form_inputs_and_urls in such scenario...